app/models/post.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class Post < ApplicationRecord
   validates :title, presence: true, length: { maximum: 50 }
   validates :body, allow_blank: true, length: { maximum: 300 }
```

Comment:
You actually do not want to allow blank post body, the allow_blank keyword there does that